package.json

```diff
@@ -97,6 +97,6 @@
     "lint-staged": "^6.0.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "semantic-release": "^8.0.3"
+    "semantic-release": "^11.0.2"
```

Comment:
Fyi v12 just dropped over the weekend. So maybe go straight to that

Reply:
Thanks, upgrade to v12, however part of the v12 migration is to ensure TravisCI build only runs once.
Either with https://github.com/semantic-release/travis-deploy-once or the new **Build Stages** beta feature.
Will need to investigate this and add a follow up commit.

Reply:
Yeah that used to happen by default. This is what we had to override on the internal build systems. So essentially you're re-enabling what was default behaviour in v11.